src/client/addon/ie11/preset.ts

```diff
@@ -0,0 +1,36 @@
+import type { Configuration } from 'webpack';
+
+const nodeModulesThatNeedToBeParsedBecauseTheyExposeES6 = ['creevey', '@testing-library'];
+
+const include = new RegExp(
+  `[\\\\/]node_modules[\\\\/](${nodeModulesThatNeedToBeParsedBecauseTheyExposeES6.join('|')})`,
```

Comment:
Does it handle only windows paths?

Reply:
No, the final regexp looks like `/[\\/]node_modules[\\/](...)/` and handles both types of slashes. So it works for Windows and Unix paths.